test/e2e/parallel/space_test.go

```diff
@@ -279,7 +276,12 @@ func TestSubSpaces(t *testing.T) {
 	t.Run("we create subSpaces in the parentSpace tree and expect roles and usernames to be inherited in NSTemplateSet", func(t *testing.T) {
 		// when
 		// we have a parentSpace
-		parentSpace, _, parentSpaceBindings := CreateSpace(t, awaitilities, testspace.WithSpecTargetCluster(memberAwait.ClusterName), testspace.WithTierName("appstudio"))
+		_, mur := CreateUserSignupWithSpaceTier(t, awaitilities, "appstudio")
```

Comment:
it's the same as in the other PR - we expect that the space is provisioned in the first member cluster, but `CreateUserSignupWithSpaceTier` can provision it anywhere